README.md

```diff
@@ -1,165 +1,20 @@
-# tudo
+# Tudo
```

Comment:
we should _really_ be re-using the _old_ readme which has/had a lot more introductory detail.